modyn/tests/trainer_server/internal/grpc/test_trainer_server_grpc_servicer.py

```diff
@@ -166,6 +166,12 @@ def test_start_training_invalid(test_hasattr, test_connect_to_model_storage):
         assert not trainer_server._training_dict
         assert trainer_server._next_training_id == 0
 
+
+@patch.object(TrainerServerGRPCServicer, "connect_to_model_storage", return_value=DummyModelStorageStub())
+@patch("modyn.trainer_server.internal.grpc.trainer_server_grpc_servicer.hasattr", return_value=True)
+def test_start_training_invalid_id(test_hasattr, test_connect_to_model_storage):
+    with tempfile.TemporaryDirectory() as modyn_temp:
+        trainer_server = TrainerServerGRPCServicer(modyn_config, modyn_temp)
```

Comment:
is that just me or does the diff look weird? it looks like you added a function header here, but where is the actual test coming from?